.docker/tests/test_aiida.py

```diff
@@ -30,3 +30,9 @@ def test_verdi_status(aiida_exec, container_user, timeout):
 
     # check that we have suppressed the warnings
     assert 'Warning' not in output
+
+
+def test_computer_setup_success(aiida_exec, container_user, timeout):
+    time.sleep(timeout)
+    output = aiida_exec('verdi computer show localhost', user=container_user).decode().strip()
```

Comment:
Should we maybe also test `verdi computer test localhost`? This checks even more

Reply:
Yes, should be better. Let's see if the test passed.